Leetcode/Leetcode_6.playground/Contents.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+func convert(_ s: String, _ numRows: Int) -> String {
```

Comment:
better add suggestion
```suggestion
// convert xxx (test PR suggestion)
func convert(_ s: String, _ numRows: Int) -> String {
```